tests/docker-compose.yml

```diff
@@ -33,9 +33,13 @@ services:
     container_name: "lizmap${LZMBRANCH}_test_php"
     environment:
       PGSERVICEFILE: /srv/etc/pg_service.conf
+      LIZMAP_ADMIN_LOGIN: admin
```

Comment:
values of these variable should be environment variable. We don't want to have to modify this file to change these values. default values should be set into run-docker.